src/collections/posts/hidden-features-of-beatsaver.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Uncovering Hidden Features of BeatSaver
+section: articles
+category: guide
+publish: 2024-05-25T00:00:00.000Z
+homepageText: Not all BeatSaver features are apparent at first sight. This
+  article reveals hidden aspects of the BeatSaver platform such as tag
+  filtering, multi-tag selection, multi-adding to bookmarks, direct map search
+  via keys, double upload prevention, review curations, and displaying bot maps.
+image: /static/uploads/posts/uncovering-beatsaber-features-cover.png
+authors:
+  - '139574'
+showInPostListing: true
+linkToSpecialtyPage: /getting-started
+---
+
+[BeatSaver](https://beatsaver.com) is the place to get and distribute maps. If you are using a custom song mod it is more than likely that it retrieves from there. Some of BeatSaver's features can be a bit hidden or undocumented or only become apparent in specific cases.
+
+The following features are from the view of "normal users" and will thus go into aspects that are relevant when you are simply using the website to view and find maps or upload your own maps.
+
+# Excluding Tags
+
+By opening the filter dropdown you gain the ability to filter for tags. For example, if you are into Tech maps, you can search for Tech maps specifically.
+
+![](/uploads/posts/uncovering-beatsaber-features-1.png)
+
+At first glance, it appears to only allow for positive filtering; meaning you can only declare "I want to only be shown maps that have the XYZ tag".
+
+But what if you are having a lazy day and want to avoid Fitbeat maps? Just press the `[Alt]` button while clicking the tag that you want to exclude. You will find that the tag turns red, and maps with such a tag will no longer appear in your search!
```

Comment:
For these keyboard keys, you *could* use the `<kbd>` tag, and give it a nice style to have it stand out from code blocks.
The [JetBrains docs](https://www.jetbrains.com/help/phpstorm/wrapping-a-tag-example-of-applying-surround-live-templates.html) do this quite nicely.